test/kanikoBuildTask/build.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: Build
+metadata:
+  name: kaniko-build
+spec:
+  source:
+    git:
+      url: https://github.com/pivotal-nader-ziada/gohelloworld
+      revision: master
+  template:
+    name: kaniko
+    arguments:
+    - name: IMAGE
+      value: gcr.io/cf-elafros-dog/kanikobuildtask
```

Comment:
im assuming this is a WIP test but if it isn't I'd like to know more about where these images come from

Reply:
Yes, I still have to work on documentation and tests. just wanted to get the PR out for any feedback